client/pom.xml

```diff
@@ -35,6 +35,10 @@
             <id>repo.jenkins-ci.org</id>
             <url>https://repo.jenkins-ci.org/public/</url>
         </repository>
+        <repository>
```

Comment:
Should not be required, it comes from upstream

Reply:
Sorry, something was glitchy in my IntelliJ/Maven setup and it wouldn't build the project at first without this section. But you are right, it is not necessary.